x-pack/plugins/monitoring/server/lib/logstash/get_nodes.test.ts

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { getNodes } from './get_nodes';
+import { STANDALONE_CLUSTER_CLUSTER_UUID } from '../../../common/constants';
+import { LegacyRequest } from '../../types';
+import sinon from 'sinon';
+
+jest.mock('../../static_globals', () => ({
+  Globals: {
+    app: {
+      config: {
+        ui: {
+          ccs: { enabled: true },
+        },
+      },
+    },
+  },
+}));
+
+describe('getNodes', () => {
+  it('ensures collapse key is present query responses', async () => {
+    const response = {};
+
+    const config = {
+      get: sinon.stub(),
```

Comment:
Any reason we're using sinon here instead of jest spies?

Reply:
When I grepped the repo for config stubs sinon seemed to be the most common option. I'll look for some jest spies.

Reply:
I couldn't find a good pre-existing example of config stub using jest, so let me know if https://github.com/elastic/kibana/pull/124201/files#diff-66bf0d22afa3a1ce70c3ef9a701cadbe64d4774a587c8b01b110e74d8db6f46bR28-R31 seems idiomatic. It passes at least :)